FreeIT/src/lesson3/Task12book.java

```diff
@@ -0,0 +1,22 @@
+package lesson3;
+
+public class Task12book {
+
+//	Найдите сумму первых n целых чисел, которые делятся на 3.
+
+	public static void main(String[] args) {
```

Comment:
Something wrong with the logic. We should define n and sum numbers starting from 1 if the is divisible by 3.
ex: n =1 => result 3, n=3 => result 18

Reply:
What's wrong? I don't understand. I use the same logic, as you wrote.

Reply:
Try to set n=3 in your code. Result should be 18.  (3+6+9)
for n=4 result should be 30 (3+6+9+12)